cmd/kubeadm/app/node/bootstrap.go

```diff
@@ -101,15 +91,16 @@ func EstablishMasterConnection(c *kubeadmapi.TokenDiscovery, clusterInfo *kubead
 		close(result)
```

Comment:
Honestly, I thought the same thing but wanted to keep the code changes to a minimum since the PR was getting big. Will do.